app/src/org/commcare/views/widgets/ComboboxWidget.java

```diff
@@ -99,6 +95,9 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
 
             @Override
             public void afterTextChanged(Editable s) {
+                if (s != null && "".equals(s.toString())) {
```

Comment:
Nitpick: `isEmpty()` can be used here

```suggestion
                if (s != null && s.toString().isEmpty()) {
```